client/my-sites/backup/is-multisite-switch.tsx

```diff
@@ -0,0 +1,58 @@
+import { WPCOM_FEATURES_BACKUPS } from '@automattic/calypso-products';
+import { FunctionComponent, ReactNode, useCallback } from 'react';
+import { useSelector } from 'react-redux';
+import QuerySiteFeatures from 'calypso/components/data/query-site-features';
+import RenderSwitch from 'calypso/components/jetpack/render-switch';
+import getFeaturesBySiteId from 'calypso/state/selectors/get-site-features';
+import isRequestingSiteFeatures from 'calypso/state/selectors/is-requesting-site-features';
+import siteHasFeature from 'calypso/state/selectors/site-has-feature';
+import { isJetpackSiteMultiSite } from 'calypso/state/sites/selectors';
+import getSelectedSiteId from 'calypso/state/ui/selectors/get-selected-site-id';
+
+type Props = {
+	trueComponent: ReactNode;
+	falseComponent: ReactNode;
+	loadingComponent?: ReactNode;
+};
+
+const IsMultiSiteSwitch: FunctionComponent< Props > = ( {
```

Comment:
It's a bit wordy, but I wonder if something like `IsMultiSiteNoPlanSwitch` or similar might be a less misleading name.